backend/user-service/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:20-alpine
+FROM node:20-alpine as base
```

Comment:
v minor issue, but i got this warning: ` FromAsCasing: 'as' and 'FROM' keywords' casing do not match (line 1)`